<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devInfoDialog</class>
 <widget class="QDialog" name="devInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>格式化设置</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>321</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>235</width>
     <height>242</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_FormatSum">
        <property name="text">
         <string>格式化字节数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_DBISum">
        <property name="text">
         <string>DBI数量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label__DBUSize">
        <property name="text">
         <string>DBU大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_DBIStart">
        <property name="text">
         <string>DBI起始扇区</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_devTotalSize">
        <property name="text">
         <string>磁盘总容量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_BOOTStart">
        <property name="text">
         <string>BOOT起始扇区</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SectorSize">
        <property name="text">
         <string>扇区大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_BBMSectorSize">
        <property name="text">
         <string>每个BBM扇区数</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_FormatSum">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_DBISum"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_DBUSize"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_DBIStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_devTotalSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BOOTStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SectorSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BBMSectorSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_curDevName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前磁盘</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>devInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>devInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
